### PR TITLE
debian: actualize licenses for 1.10

### LINKED ISSUE #6391: debian: revisit the list of licenses in debian/copyright
Version: 2.10.0-beta1-1-gfb7fe96b5.

We should revisit / update the list of licenses in `debian/copyright`. Looked briefly:

* I don't see any entry about third_party/luarocks.
* The same for third_party/c-ares (what if we'll make it conditional within #6390?)
* The same for third_party/curl.
* The same for third_party/decNumber.

We should carefully review this file.

### PR DESCRIPTION
**Pull request by DifferentialOrange**
Add missing third party licenses to debian/copyright. Update copyright
dates for modules. Remove license entries for unused modules.

Closes #6391

### CODE REVIEW

File: debian/copyright
Comment (Totktonada) [inline debian/copyright:138]: Same as in PR #6542: I would fix it in checks and them there.

**Review by Totktonada — APPROVED** [review]
LGTM after checks fix.

**Comment by DifferentialOrange** [comment]
checks copyright problem was fixed in last commit.

File: debian/copyright
Comment (DifferentialOrange) [reply debian/copyright]: Fixed

**Comment by Totktonada** [comment]
I'm ok. It would be good to add 'Backported from commit \<git commit hash from master\>' after PR #6542 merge.

**Comment by DifferentialOrange** [comment]
> I'm ok. It would be good to add 'Backported from commit <git commit hash from master>' after PR #6542 merge.

Added "backported from" and "cherry picked from" comments

**Review by kyukhin — APPROVED** [review]

**Pull request merged**
